src/index.html

```diff
@@ -13,7 +13,85 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit(event)">
+    <fieldset class="block">
+      <legend>Personal information</legend>
+      <label>
+        Surname:
+        <input type="text" name="Surname" autocomplete="off" required>
+      </label>
+      <label>
+        Name:
+        <input type="text" name="Name" autocomplete="off" required>
+      </label>
+      <label>
+        How old are You?
+        <input type="number" name="ages" value="12" min="1" max="130" data-qa="field-value">
+      </label>
+      <label>
+        Full date of birth:
+        <input type="date" name="date of birth">
```

Comment:
better to write without whitespaces
```suggestion
        <input type="date" name="date_of_birth">
```